rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -869,32 +860,19 @@ type StrikeSchedule extends RateSchedule: <"A class describing a schedule of cap
     buyer PayerReceiverEnum (0..1) <"The buyer of the option.">
     seller PayerReceiverEnum (0..1) <"The party that has sold.">
 
-type CalculationAgentModel: <"This class corresponds to the FpML CalculationAgent.model.">
-    [deprecated]
-
-    calculationAgent CalculationAgent (0..1) <"The ISDA calculation agent responsible for performing duties as defined in the applicable product definitions.">
-    calculationAgentBusinessCenter BusinessCenterEnum (0..1) <"The city in which the office through which ISDA Calculation Agent is acting for purposes of the transaction is located. The short-form confirm for a trade that is executed under a Sovereign or Asia-Pacific Master Confirmation Agreement (MCA), does not need to specify the Calculation Agent. However, the confirm does need to specify the Calculation Agent city. This is due to the fact that the MCA sets the value for Calculation Agent but does not set the value for Calculation Agent city.">
-
 type AssetPayout extends PayoutBase: <"Security finance payout specification in case the product payout involves some form of security collateral, as in a securities financing transaction. Plus additional description for ICMA.">
     [metadata key]
 
     assetLeg AssetLeg (1..*) <"Defines each asset movement as a buy/sell at different dates, typically 1 near leg and 1 far leg in a securities financing transaction.">
-    securityInformation Product (1..1) <"Specifies the Purchased Security.  Within SecurityPayout we include a condition which validates that the product must be a Security (see below condition 'ProductMustBeSecurity').">
-    durationType Duration (1..1) <"Specifies the Duration Terms of the Security Finance transaction. e.g. Open or Term.">
-        [deprecated]
+    securityInformation Security (1..1) <"Specifies the Purchased Security.">
```

Comment:
Idea: (i) change type to `Asset` and (ii) rename attribute as `underlier`.

(i) Payout is meant to represent synthetic asset economics (including the asset's initial and final transfer) typically used in an asset financing trade. The reason it was narrowed down to `Security` before is because `Asset` did not exist.
(ii) would further alignment between different payout types to use the same attribute name.

Reply:
done

Added a condition `underlier -> Cash is absent`

Changed instances of
`assetPayout -> securityInformation -> identifier`
to
`assetPayout -> underlier ->> identifier`

@Hugo IMO this will result in mapping changes